Examples/destination_plugins/AppsFlyerDestination.swift

```diff
@@ -0,0 +1,352 @@
+                "source": deepLinkObj.mediaSource ?? "",
+                "name": deepLinkObj.campaign ?? "",
+                "product": deepLinkObj.deeplinkValue ?? ""
+            ]
+            
+            let properties: [String: Any] = [
+                "provider": "AppsFlyer",
+                "campaign": campaign
+            ]
+            
+            analytics?.track(name: "Deferred Deep Link", properties: properties)
+            
+        } else {
+            
+            let campaign: [String: Any] = [
+                "source": deepLinkObj.mediaSource ?? "",
+                "name": deepLinkObj.campaign ?? "",
+                "product": deepLinkObj.deeplinkValue ?? ""
+            ]
+            
+            let properties: [String: Any] = [
+                "provider": "AppsFlyer",
+                "campaign": campaign
+            ]
+            
+            analytics?.track(name: "Direct Deep Link", properties: properties)
+            
+        }
+        
+        //Logic to grab AppsFlyer's deep link value to instantiate correct VC
+        //        guard let productNameStr = deepLinkObj.deeplinkValue else {
+        //            print("Could not extract deep_link_value from deep link object")
+        //            return
+        //        }
+        
+        //implement your own logic to open the correct screen/content
+        //        walkToSceneWithParams(product: productNameStr, deepLinkObj: deepLinkObj)
+    }
+    
+    
+    // User logic for opening Deep Links
+    //    fileprivate func walkToSceneWithParams(product: String, deepLinkObj: DeepLink) {
+    //        let storyBoard: UIStoryboard = UIStoryboard(name: "Main", bundle: nil)
+    //        UIApplication.shared.windows.first?.rootViewController?.dismiss(animated: true, completion: nil)
+    //
+    //        let destVC = "product_vc"
+    //        if let newVC = storyBoard.instantiateVC(withIdentifier: destVC) {
+    //
+    //            print("[AFSDK] AppsFlyer routing to section: \(destVC)")
+    //            newVC.deepLinkData = deepLinkObj
+    //
+    //            UIApplication.shared.windows.first?.rootViewController?.present(newVC, animated: true, completion: nil)
+    //        } else {
+    //            print("[AFSDK] AppsFlyer: could not find section: \(destVC)")
+    //        }
+    //    }
+}
+
+//MARK: - UI StoryBoard Extension; Deep Linking
+
+//Aditonal logic for Deep Linking
+//extension UIStoryboard {
+//    func instantiateVC(withIdentifier identifier: String) -> DLViewController? {
+//        // "identifierToNibNameMap" – dont change it. It is a key for searching IDs
+//        if let identifiersList = self.value(forKey: "identifierToNibNameMap") as? [String: Any] {
+//            if identifiersList[identifier] != nil {
+//                return self.instantiateViewController(withIdentifier: identifier) as? DLViewController
+//            }
+//        }
+//        return nil
+//    }
+//}
```

Comment:
what's all this commented out code for?

Reply:
because it's for deep linking, requires the user to adjust it, and isn't necessary for everything else to work. We say in the current AppsFlyer dest that deep linking isn't supported out of the box so this is following that trend.